app/ui-react/syndesis/src/modules/data/pages/viewCreate/SelectNamePage.tsx

```diff
@@ -36,122 +38,153 @@ export class SelectNamePage extends React.Component {
-                        definition={definition}
-                        initialValue={{
-                          description: '',
-                          name: '',
-                        }}
-                        validate={validate}
-                        onSave={onSave}
-                      >
-                        {({
-                          fields,
-                          handleSubmit,
-                          isSubmitting,
-                          isValid,
-                          submitForm,
-                        }) => (
-                          <ViewCreateLayout
-                            header={<ViewCreateSteps step={2} />}
-                            content={
-                              <>
-                                <PageTitle
-                                  title={i18n.t(
-                                    'data:virtualization.createViewWizardSelectNameTitle'
-                                  )}
-                                />
-                                <ViewConfigurationForm
-                                  i18nFormTitle={i18n.t(
-                                    'data:virtualization.createViewWizardSelectNameTitle'
-                                  )}
-                                  handleSubmit={handleSubmit}
+                <WithRouteData<ISelectNameRouteParams, ISelectNameRouteState>>
+                  {(
+                    { virtualizationId },
+                    { virtualization, schemaNodeInfo },
+                    { history }
+                  ) => (
+                    <WithVirtualizationHelpers>
+                      {({ refreshVirtualizationViews }) => {
+                        const onSave = async (
+                          { name, description }: ISaveForm,
+                          actions: any
+                        ) => {
+                          // ViewEditorState for the source
+                          const viewEditorState = generateViewEditorState(
+                            virtualization.serviceVdbName,
+                            schemaNodeInfo,
+                            name,
+                            description
+                          );
+                          try {
+                            await refreshVirtualizationViews(
+                              virtualization.keng__id,
+                              [viewEditorState]
+                            );
+                            pushNotification(
+                              t('virtualization.createViewSuccess', {
+                                name: viewEditorState.viewDefinition.viewName,
+                              }),
+                              'success'
+                            );
+                          } catch (error) {
+                            const details = error.message ? error.message : '';
+                            pushNotification(
+                              t('virtualization.createViewFailure', {
```

Comment:
doesnt match translations - should be 'createViewFailed' not 'createViewFailure'